app/controllers/v0/form1095_bs_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module V0
+  class Form1095BsController < ApplicationController
+    before_action { authorize :form1095, :access? }
+    before_action :get_1095b, only: :download
+    before_action :get_available_forms, only: :available_forms
+
+    def available_forms
+      render json: { available_forms: @available_forms_arr }
+    end
+
+    def download
+      file_name = "1095B_#{download_params}.pdf"
```

Comment:
I guess when only having one required param, the param becomes that value (instead of a hash). Using download_params[:tax_year] errors out, because download_params only has the tax year in it (i.e. download_params == 2021)

Reply:
wait disregard, found your .permit comment below finally